plugin/cache/cache.go

```diff
@@ -174,7 +174,7 @@ func (w *ResponseWriter) WriteMsg(res *dns.Msg) error {
 
 	if hasKey && duration > 0 {
 		if w.state.Match(res) {
-			w.set(res, key, mt, duration)
+			w.set(res, key, mt, duration, w.prefetch)
```

Comment:
shouldn't `w.prefetch` be available in set as it's a method on `w` ?

Reply:
oh duh.  thanks!